### ISSUE TITLE
Composer installation complaining about package minimun stability

### ISSUE THREAD

**Issue by mrmedina87** [multidots/zf3-rest-api#2]
I could not install it using composer; the results shows the following:

Could not find package multidots/zf3-rest-api at any version for your minimum-stability (stable). Check the package  spelling or your minimum-stability

**Comment by narendravaghela** [comment]
Hi @mrmedina87 
The stable version of this package is not released yet. Please try to remove the minimum-stability check from your composer.json file or set it to 'dev'.

**Comment by mrmedina87** [comment]
Hello @narendravaghela I have other question: in order to install this package, should I install all the zend 3 framework first? 

I mean, using the command: 
composer require zendframework/zendframework
And after that: composer require multidots/zf3-rest-api

If that's the case, I do not understand this:

Now copy this file "vender/multidots/zf3-rest-api/config/restapi.global.php" and paste to root "config/autoload/restapi.global.php" 

I do not have a "config/autoload/" folder. I am using a Windows file system. I did not find the file either modules.config.php.

Thanks.

**Comment by ankitgs** [comment]
Yes, this is a ZF3 module. So, you first need to install ZF3 framework and then you can install and use this module.

**Comment by ankitgs** [comment]
Hello @mrmedina87 

For setup a ZF3 project Please refer the following link.

> https://docs.zendframework.com/tutorials/getting-started/skeleton-application/
